multiqc/utils/search_patterns.yaml

```diff
@@ -345,6 +345,10 @@ flash/hist:
 flexbar:
   contents: "Flexbar - flexible barcode and adapter removal"
   shared: true
+freyja:
+  fn: "*.tsv"
+  contents: "summarized"
```

Comment:
Feels not very specific - do you think more conditions can be added? Is it fair to expect that files are named `*demix_outs.tsv`?

Reply:
I guess not... 
Perhaps we could check more contents lines? Like require `lineages` and `abundances`?

Reply:
Yeah definitely if its better to add those checks. I thought it would be enough because of the number of lines we specify.